### ISSUE TITLE
"No declaration found for hb_glyph_info_t"

### ISSUE THREAD

**Issue by SimonSapin** [harfbuzz/harfbuzz#1825]
`make dist` prints (among other output):

```
./harfbuzz-sections.txt:101: warning: No declaration found for hb_glyph_info_t.
```

Indeed, `hb_glyph_info_t` is not a link when mentioned in https://harfbuzz.github.io/harfbuzz-hb-buffer.html, whereas `hb_glyph_position_t` for example is. I don’t understand more than that, though.

(There are other docs warnings too, this just happens to be the one for a struct I was looking for specifically.)

**Comment by behdad** [comment]
Yeah I don't know why it doesn't pick it up from `hb-buffer.h`.

> (There are other docs warnings too, this just happens to be the one for a struct I was looking for specifically.)

We should look into most of those, yes.

**Comment by khaledhosny** [comment]
This has been fixed in b8b13018edc4115e9857d7871d254c5737f90046
